code/modules/clothing/clothing.dm

```diff
@@ -397,13 +400,16 @@
 // you just dont get the same feeling with handwashed clothes
 /obj/item/clothing/machine_wash()
 	. = ..()
-	var/fresh_mood = AddComponent( \
-		/datum/component/onwear_mood, \
-		saved_event_type = /datum/mood_event/fresh_laundry, \
-		examine_string = "[src] looks crisp and pristine.", \
-	)
+	if(stubborn_stains) //Just can't make it feel right
+		return
+	else
+		var/fresh_mood = AddComponent( \
+			/datum/component/onwear_mood, \
+			saved_event_type = /datum/mood_event/fresh_laundry, \
+			examine_string = "[src] looks crisp and pristine.", \
+		)
 
-	QDEL_IN(fresh_mood, 2 MINUTES)
+		QDEL_IN(fresh_mood, 2 MINUTES)
```

Comment:
just keep `if(stubborn_stains)` as an early return, no need to quirk out the indentation by having the `else` case.

```dm
	if(stubborn_stains) //Just can't make it feel right
		return

	var/fresh_mood = AddComponent( \
		/datum/component/onwear_mood, \
		saved_event_type = /datum/mood_event/fresh_laundry, \
		examine_string = "[src] looks crisp and pristine.", \
	)

	QDEL_IN(fresh_mood, 2 MINUTES)
```